content/en/serverless/aws_lambda/distributed_tracing.md

```diff
@@ -108,15 +110,24 @@ For .NET serverless applications, Datadog recommends [installing Datadog's traci
 
 Learn more about [tracing through .NET Azure serverless applications][15].
 
-### Hybrid environments
+## Span Auto-linking
+Datadog can automatically detect linked spans when certain parts of your request cannot propogate trace context, such as when a request triggers an [S3 Change Events][28], or [DynamoDB Streams][29]. These will appear in the Span Links tab in the Datadog Trace UI. 
+
+**Note:** Span auto-linking may not work if you are only ingesting a sample of your traces because the linked traces might be dropped before ingestion. To improve your chances of seeing auto-linked spans, increase your sample rate. 
```

Comment:
There is also the matter of not having the upstream span pointers if the object or item was created without them.